### PR TITLE
Name consistency - set to true if neither have any names.

### PR DESCRIPTION
**Pull request by dnesbitt61**

### CODE REVIEW

**Comment by gknisely** [comment]
:ship:

File: src/mjolnir/graphenhancer.cc [hunk 1]
```diff
@@ -905,6 +905,13 @@ bool ConsistentNames(const std::string& country_code,
...
   std::unique_ptr<StreetNames> street_names2 = StreetNamesFactory::Create(
       country_code, names2);
+
+  // Flag as consistent names when neither has names!
+  if (street_names1->empty() && street_names2->empty()) {
+    return true;
+  }
+
+  // Return true (consistent) if the common base names are not empty
   return (!(street_names1->FindCommonBaseNames(*street_names2)->empty()));
```
Comment (kevinkreiser) [inline src/mjolnir/graphenhancer.cc]: this shouldn't go into the FindCommonBaseNames function?

Comment (dgearhart) [inline src/mjolnir/graphenhancer.cc]: Find common base names is specific to named roads - i think the the unnamed case is separate

**Comment by dgearhart** [comment]
:shipit:

File: src/mjolnir/graphenhancer.cc (hunk 1 above)
Comment (kevinkreiser) [inline src/mjolnir/graphenhancer.cc]: fair enough sounds good to me

**Pull request merged**